discovery_engine_core/core/src/storage/migrations/20220727110259_stack_id.sql

```diff
@@ -0,0 +1,30 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE Stack(
```

Comment:
❓ why doesn't this table and the `StackDocument` table need an `IF NOT EXISTS`?

Reply:
good point they need it as much as any other table in our migrations :sweat_smile: 

I will add it here but we might want to discuss this in slack.

Mainly do we want a migration which is run twice to:
- error
- not error, but use the tables of the first run, which might not be the same if you change a migration